docs/tutorials/from-tabular-data-to-binned-data.ipynb

```diff
@@ -0,0 +1,679 @@
+    "The histogrammed data consists of essentially two arrays, one for the values (yellow) and one for the wavelengths (green):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "74ce6bf9-7ef7-4a1f-ac7a-46fd6f752815",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.show(histogrammed)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ceddb439-6643-4243-aaef-e15408d4d75f",
+   "metadata": {},
+   "source": [
+    "The top level structure of the binned data is the same, i.e., we have and array of values and an array of wavelengths.\n",
+    "The difference is that each value (bin) stores all contributing table rows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc4c3303-537a-4ea1-908c-5db7d77e7f2d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.show(binned)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e83a97e1-4e07-45bd-8ad2-7dd66db65c69",
+   "metadata": {},
+   "source": [
+    "### Exercise 1\n",
+    "\n",
+    "- Define bin edges for `z_pos` and use it to histogram and bin `table`.\n",
+    "  Plot the results.\n",
+    "- Define different bin edges for `z_pos`, e.g., with more values.\n",
+    "  Use `sc.bin` with the new edges *on the result of the binning from the first bullet*, i.e., *not* on the original table `table`.\n",
+    "  Why is this possible?\n",
+    "  \n",
+    "#### Solution"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c8c0ccb7-65fa-4fd8-8c7c-2782e25207ec",
+   "metadata": {
+    "tags": [
+     "solution"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "z = table.coords['z_pos']\n",
+    "z_pos = sc.linspace('z_pos', z.min().value, z.max().value, num=101, unit='mm')\n",
+    "binned = sc.bin(table, edges=[z_pos])\n",
+    "solution1 = dict()\n",
+    "solution1['histogrammed'] = sc.histogram(table, bins=z_pos)\n",
+    "solution1['binned'] = binned.bins.sum()\n",
+    "z_pos = sc.linspace('z_pos', z.min().value, z.max().value, num=301, unit='mm')\n",
+    "solution1['binned_high_resolution'] = sc.bin(binned, edges=[z_pos]).bins.sum()\n",
+    "sc.plot(solution1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96f30fb7-1ff3-45c3-bb3b-21a23ddd08e6",
+   "metadata": {},
+   "source": [
+    "## Multi-dimensional spatial binning"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ee28604-8ec5-46bf-a9a9-26667eb7f314",
+   "metadata": {},
+   "source": [
+    "`sc.bin` can handle multiple dimensions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97965f28-1bd4-45dd-856a-2b55d0d53894",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = table.coords['x_pos']\n",
+    "y = table.coords['y_pos']\n",
+    "z = table.coords['z_pos']\n",
+    "x_pos = sc.linspace('x_pos', x.min().value, x.max().value, num=31, unit='mm')\n",
```

Comment:
Same as above, rename to clarify that these are edges and not the same as the stored coord.